src/gluonts/shell/__main__.py

```diff
@@ -77,14 +77,39 @@ def cli() -> None:
     type=click.Path(exists=True),
     envvar="SAGEMAKER_DATA_PATH",
     default='/opt/ml',
+    help='The root path of all folders mounted by the SageMaker runtime.',
 )
-@click.option("--forecaster", metavar="NAME", envvar="GLUONTS_FORECASTER")
-def serve_command(data_path: str, forecaster: Optional[str]) -> None:
+@click.option(
+    "--forecaster",
+    metavar='NAME',
+    envvar="GLUONTS_FORECASTER",
+    help=(
+        'An alias or a fully qualified name of a Predictor to use. '
+        'If this value is defined, the inference server runs in the '
+        'so-called dynamic mode, where the predictor is initialized for '
+        'each request using parameters provided by the in the "configuration" '
+        'field of the JSON request. Otherwise, the server runs in static '
+        'mode, where the predictor is initialized upfront from a '
+        'serialized model located in the {data-path}/model folder.'
+    ),
+)
+@click.option(
+    "--force-static/--no-force-static",
+    envvar="GLUONTS_FORCE_STATIC",
+    default=False,
+    help=(
+        'Forces execution in static mode, even in situations where the '
```

Comment:
For a moment I thought static mode is when the `--forecaster` option is set. Maybe the names are a bit confusing and we should think about better ones in the future.